### PR TITLE
DOCS-12265-revise mutual authentication page

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@vli63127 @sarjeet2013 @kmendiratta-crm , I revised this page to include the API curl command and to restructure it a bit so anyone on an earlier version hit /latest/ could decide whether or not it was applicable

ideally we could replace 7/15/2020 with an actual version number?

### CODE REVIEW

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 1]
```diff
@@ -3,7 +3,16 @@ ifndef::env-site,env-github[]
...
 
-With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 15, 2020 have the mutual authentication setting enabled by default.
+[IMPORTANT]
+--
+Mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
+--
+
+Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. 
+
+If you're on aRuntime Fabric version earlier than 1.6.45, you can enable mutual authentication using the Runtime Fabric API. Or, on Anypoint Runtime Fabric version 1.3.60 or later, you can use Runtime Manager to enable it.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
If you're on a Runtime Fabric version earlier than 1.6.45, you can enable mutual authentication using the Runtime Fabric API. Or, on Anypoint Runtime Fabric version 1.3.60 or later, you can use Runtime Manager to enable it.
```

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 2]
```diff
@@ -3,7 +3,16 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 15, 2020 have the mutual authentication setting enabled by default.
+[IMPORTANT]
+--
+Mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
Mutual authentication for connectivity between Runtime Fabric and Anypoint control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
```

**Review by sarjeet2013 — DISMISSED** [review]
lgtm

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 3]
```diff
@@ -3,7 +3,16 @@ ifndef::env-site,env-github[]
...
 
-With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 15, 2020 have the mutual authentication setting enabled by default.
+[IMPORTANT]
+--
+Mutual authentication for connectivity between Runtime Fabric and Anypoint control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
+--
+
+Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. 
+
+If you're on a Runtime Fabric version earlier than 1.6.45, you can enable mutual authentication using the Runtime Fabric API. Or, on Anypoint Runtime Fabric version 1.3.60 or later, you can use Runtime Manager to enable it.
```
Comment (hannanelson) [inline modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
For Runtime Fabric versions earlier than 1.6.45, enable mutual authentication using the Runtime Fabric API. For Runtime Fabric versions 1.3.60 and later, use Runtime Manager to enable it.
```

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 4]
```diff
@@ -22,3 +30,23 @@ cd /opt/anypoint/runtimefabric
 ./rtfctl update
 ./rtfctl test outbound-network --mutual-tls-check
 ----
+
+== Enable Mutual Authentication from Runtime Manager
+
+. From Runtime Manager, select the appropriate environment and click Runtime Fabrics.
```
Comment (hannanelson) [inline modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
. From Runtime Manager, select the appropriate environment and click *Runtime Fabrics*. 
```

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 5]
```diff
@@ -12,8 +21,7 @@ Before enabling mutual authentication, verify the following endpoint and port ar
 * EU control plane:
 ** `configuration-resolver.prod-eu.msap.io:443`
 
-[IMPORTANT]
-Failure to meet the previously described connectivity requirements before enabling mutual authentication results in errors with creating/managing application deployments in Runtime Fabric.
+Failure to meet these requirements before enabling mutual authentication results in errors with creating and managing application deployments in Runtime Fabric.
```
Comment (hannanelson) [inline modules/ROOT/pages/config-mutual-auth.adoc]: Is it "in" or "on"?

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 6]
```diff
@@ -3,7 +3,16 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 15, 2020 have the mutual authentication setting enabled by default.
+[IMPORTANT]
+--
+Mutual authentication for connectivity between Runtime Fabric and Anypoint control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
```
Comment (hannanelson) [inline modules/ROOT/pages/config-mutual-auth.adoc]: Not sure about "Anypoint control plane". Correct?

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/config-mutual-auth.adoc (hunk 5 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
Failure to meet these requirements before enabling mutual authentication results in errors with creating and managing application deployments on Runtime Fabric.
```

File: modules/ROOT/pages/config-mutual-auth.adoc (hunk 6 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
Mutual authentication for connectivity between Runtime Fabric and the Anypoint Platform control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
```

File: modules/ROOT/pages/config-mutual-auth.adoc [hunk 7]
```diff
@@ -3,7 +3,16 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-With Anypoint Runtime Fabric version 1.3.60 or later, you can use the Runtime Fabric *Settings* page in Runtime Manager to enable mutual authentication for connectivity between the Runtime Fabric and Anypoint control plane. Mutual authentication improves the security of communication between Runtime Fabric and Anypoint Platform and is highly recommended. Runtime Fabric instances created after July 15, 2020 have the mutual authentication setting enabled by default.
+[IMPORTANT]
+--
+Mutual authentication for connectivity between Runtime Fabric and the Anypoint Platform control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication are relevant only to those versions earlier than 1.6.45.
```
Comment (hannanelson) [inline modules/ROOT/pages/config-mutual-auth.adoc]:
```suggestion
Mutual authentication for connectivity between Runtime Fabric and the Anypoint Platform control plane is enabled by default for Runtime Fabric versions 1.6.45 and later. The following instructions to enable mutual authentication apply only to those versions earlier than 1.6.45.
```

**Review by hannanelson — DISMISSED** [review]

**Review by hannanelson — APPROVED** [review]

**Review by hannanelson — APPROVED** [review]

**Pull request merged**